docs/_data/portland-2020-config.yaml

```diff
@@ -130,6 +130,7 @@ sponsors:
     - name: microsoft
       link: https://microsoft.com
       brand: Microsoft
+      text: Our mission is to empower every person and organization on the planet to achieve more.  Come build your skills with Microsoft Learn and Microsoft Docs, where you'll find high-quality content and connect with a vibrant community.  Microsoft is a distributed company with documentarians working around the globe. Visit <a href="https://careers.microsoft.com/">https://careers.microsoft.com</a> to learn about opportunities.
```

Comment:
We should probably render this through markdown or rst, but didn't have a filter handy for it.